tests/queries/0_stateless/01269_create_with_null.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS data_null;
+
+CREATE TABLE data_null (
+    a INT NULL,
```

Comment:
What will be written into metadata file in this case? If it will be something like `a Nullable(INT) NULL`, then server will fail with `Cant use NULL with Nullable` on restart. It worth to add `SHOW CREATE` to this test. Also behaviour with  `data_type_default_nullable=1` should be tested